_templates/exchanges.html

```diff
@@ -22,6 +22,7 @@ <h4>Note: Exchanges provide highly varying degrees of safety, security,
       <h3 id="international"><img src="/img/flags/ALL.png" alt="Globe">International</h3>
       <p>
         <a href="https://bisq.network/">Bisq</a><br>
+        <a href="https://bitflyer.com/en-us/">bitFlyer</a><br>
```

Comment:
Please change to `Bitflyer`. We have a policy to not adhere to special capitalization formats for brand names (e.g. TREZOR is listed as Trezor). Also, you all are just in Japan and in the United States, no? If so, can you please update this and add yourself to the United States, category, instead of the international category?

Reply:
Launched in US, Japan, and EU, able to service customers internationally.